staging/src/k8s.io/apimachinery/pkg/apis/meta/v1alpha1/types.go

```diff
@@ -52,6 +52,10 @@ type TableColumnDefinitions struct {
 	Format string `json:"format"`
 	// description is a human readable description of this column.
 	Description string `json:"description"`
+	// priority is an integer defining the relative importance of this column compared to others. Lower
+	// numbers are considered higher priority. Columns that may be omitted in limited space scenarios
+	// should be given a higher priority.
+	Priority int32 `json:"priority"`
```

Comment:
I can't serialize tables to protobuf because of `interface{}` - I don't think we need to do protobuf for 1.7, but I'll do it in the follow up.  I'm not actually sure how to solve cells yet for protobuf.